conf/general.yml-example

```diff
@@ -114,6 +114,7 @@ GEOCODER: ''
 # relevant key here.
 GOOGLE_MAPS_API_KEY: ''
 BING_MAPS_API_KEY: ''
+GOOGLE_MAPS_JS_API_KEY: ''
```

Comment:
`GOOGLE_MAPS_API_KEY` is used in the Perl for the geocoder stuff, whereas this is solely for the maps JS. [The docs](https://developers.google.com/maps/documentation/javascript/get-api-key) made me think they'd need to be two separate keys but happy to be proven wrong if it removes the need for an extra config key!